### PR TITLE
Prepare for release, remove default hardware acceleration in example

### PR DESCRIPTION
**Pull request by gozmanyoni**

### CODE REVIEW

File: examples/carplay-web-app/src/worker/render/lib/utils.ts [hunk 1]
```diff
@@ -46,7 +46,6 @@ function getDecoderConfig(frameData: Uint8Array): VideoDecoderConfig | null {
       codec: sps.MIME,
       codedHeight: sps.picHeight,
       codedWidth: sps.picWidth,
-      hardwareAcceleration: 'prefer-hardware',
```
Comment (gozmanyoni) [inline examples/carplay-web-app/src/worker/render/lib/utils.ts:49]: Removing this as default

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-carplay",
-  "version": "4.0.0",
```
Comment (gozmanyoni) [inline package.json:3]: Bumping for release

**Comment by rhysmorgan134** [comment]
Looks good to me, removing the HW default is ideal for my setup, out of curiosity what was the reason behind changing it? Pi5 compatibility?

**Comment by gozmanyoni** [comment]
@rhysmorgan134 Yes- removing it for pi5 compatibility, also its misleading, it wont prefer hardware, it will either use hardware of throw an error and fail, which is misleading.

**Pull request merged**